### ISSUE TITLE
Inline schema's "default", "maximum"... attributes are discarded by parser (or model?)'s ModelImpl class

### ISSUE THREAD

**Issue by QianJin2013** [swagger-api/swagger-parser#239]
When define following parameter with a simple inline schema

```
...
  post:
      ... ...
      parameters:
        - name: amount
          in: body
          schema:
            type: integer
            format: int64
            description: amount of money
            default: 1000
            maximum: 100000
```

I use swagger parser to read the yaml,  it is strange the "default", "maximum"... attributes are disappeared in its ModelImpl instance!

<img width="406" alt="2016-04-29 16 08 54" src="https://cloud.githubusercontent.com/assets/5234060/14910012/b8dcdebc-0e24-11e6-9b30-ac9d14c241a4.png">

Though this is not a big problem, i can have other way to define this attributes, but it's better to let you know this so fix it later.

**Comment by QianJin2013** [comment]
by the way, "maxItems", "minItems" always discarded by ArrayProperty

**Comment by rdaviscc** [comment]
I am experiencing the same issue, only with the ArrayProperty minItems and maxItems

**Comment by rdaviscc** [comment]
My workaround currently:
Reading the map separately with Jackson's object mapper and using the Property Deserializer to account for these values.  Works but would be much better if it worked out of the box.

**Comment by fehguy** [comment]
Yea, that's unfortunate.  I will get this addressed before release.

**Comment by fehguy** [comment]
ok some bad news on this issue.  The `ModelImpl` does not have those properties, so they cannot be set.  Likewise the `ArrayModel` does not have `maxItems`, `minItems` either.

**Comment by benken-parasoft** [comment]
Also see https://github.com/swagger-api/swagger-core/issues/1201#issuecomment-155935359

**Comment by fehguy** [comment]
Fixed with #364
